[2025-02-18T13:47:39.902-0600] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-18T13:47:39.918-0600] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: FirstDAG.Firsttask scheduled__2025-02-18T00:10:00+00:00 [queued]>
[2025-02-18T13:47:39.930-0600] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: FirstDAG.Firsttask scheduled__2025-02-18T00:10:00+00:00 [queued]>
[2025-02-18T13:47:39.931-0600] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-18T13:47:39.955-0600] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): Firsttask> on 2025-02-18 00:10:00+00:00
[2025-02-18T13:47:39.960-0600] {standard_task_runner.py:72} INFO - Started process 92884 to run task
[2025-02-18T13:47:39.976-0600] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'FirstDAG', 'Firsttask', 'scheduled__2025-02-18T00:10:00+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/dags.py', '--cfg-path', '/tmp/tmpv_dtmejy']
[2025-02-18T13:47:39.981-0600] {standard_task_runner.py:105} INFO - Job 15: Subtask Firsttask
[2025-02-18T13:47:40.066-0600] {task_command.py:467} INFO - Running <TaskInstance: FirstDAG.Firsttask scheduled__2025-02-18T00:10:00+00:00 [running]> on host IMNOTROY.
[2025-02-18T13:47:40.179-0600] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='FirstDAG' AIRFLOW_CTX_TASK_ID='Firsttask' AIRFLOW_CTX_EXECUTION_DATE='2025-02-18T00:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-18T00:10:00+00:00'
[2025-02-18T13:47:40.181-0600] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-18T13:47:40.185-0600] {python.py:240} INFO - Done. Returned value was: None
[2025-02-18T13:47:40.192-0600] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-18T13:47:40.193-0600] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=FirstDAG, task_id=Firsttask, run_id=scheduled__2025-02-18T00:10:00+00:00, execution_date=20250218T001000, start_date=20250218T194739, end_date=20250218T194740
[2025-02-18T13:47:40.269-0600] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-18T13:47:40.284-0600] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-18T13:47:40.285-0600] {local_task_job_runner.py:245} INFO - ::endgroup::
